src/uu/install/src/install.rs

```diff
@@ -737,7 +740,21 @@ fn copy(from: &Path, to: &Path, b: &Behavior) -> UResult<()> {
     } else if let Err(err) = fs::copy(from, to) {
         return Err(InstallError::InstallFailed(from.to_path_buf(), to.to_path_buf(), err).into());
     }
+    Ok(())
+}
 
+/// Strip a file using an external program.
+///
+/// # Parameters
+///
+/// * `to` - The destination file path.
+/// * `b` - The behavior configuration.
+///
+/// # Returns
+///
+/// Returns an empty Result or an error in case of failure.
+///
+fn strip_file(to: &Path, b: &Behavior) -> UResult<()> {
     if b.strip && cfg!(not(windows)) {
```

Comment:
I would move `if b.strip` to the `copy` function and replace `cfg!(not(windows))` with `#[cfg(not(windows))]`.